tests/unit/askPresentIdeas.test.ts

```diff
@@ -42,6 +48,7 @@ const constants = vi.hoisted(() => ({
 }));
 
 describe("Daily cron", () => {
```

Comment:
This should be askPresentIdeas or the test should rename to cron test maybe, but thats just a suggestion.

Reply:
Ur right, renamed the file.